src/webcomponents/commons/variant-modal-ontology.js

```diff
@@ -278,33 +301,34 @@ export default class VariantModalOntology extends LitElement {
-                                    <div class="col-md-6 ontology-tree">
-                                        ${this.rootTree[0].nodes.map(node => this.drawNode(node))}
+                                    <div class="col-md-6 ontology-tree" style="padding-bottom:10px">
+                                        ${this.rootTree?.nodes.map(node => this.drawNode(node, false))}
                                     </div>
                                     <div class="col-md-6">
                                         ${this.selectedItem ? html`
                                             <ul class="list-group infoHpo">
-                                                <li class="list-group-item"><strong>Label: </strong>${this.selectedItem.text}</li>
-                                                <li class="list-group-item"><strong>Short form: </strong>${this.selectedItem.short_form}</li>
-                                                <li class="list-group-item"><strong>Obo Id: </strong>${this.selectedItem.obo_id}</li>
-                                                <li class="list-group-item"><strong>IRI: </strong>${this.selectedItem.iri}</li>
-                                                <li class="list-group-item"><strong>Description: </strong>${this.selectedItem.description}</li>
-                                                <li class="list-group-item">
-                                                    <button type="button" class="btn btn-default btn-small ripple" @click="${e => this.addTerm(this.selectedItem.obo_id)}">Add Term</button>
+                                                <li class="list-group-item"><strong>Name: </strong>${this.selectedItem.name} </li>
+                                                <li class="list-group-item"><strong>Id: </strong>${this.selectedItem.obo_id}</li>
+                                                <li class="list-group-item"><strong>IRI: </strong>
+                                                    <a href="${BioinfoUtils.getOboLink(this.selectedItem.obo_id)}" target="_blank">
+                                                        ${BioinfoUtils.getOboLink(this.selectedItem.obo_id)}
+                                                    </a>
                                                 </li>
+                                                <li class="list-group-item"><strong>Synonyms: </strong>${this.selectedItem.synonyms?.join(", ")}</li>
+                                                <li class="list-group-item"><strong>Description: </strong>${this.selectedItem.description}</li>
+                                                <li class="list-group-item"><strong>Comment: </strong>${this.selectedItem?.comment}</li>
                                             </ul>
+                                            <button type="button" class="btn btn-default btn-small" @click="${() => this.addTerm(this.selectedItem.obo_id)}">Add Term</button>
```

Comment:
I think this part will be more readable if you put the child elements in a new line. For example:

```html
<li class="list-group-item">
    <strong>Name: </strong>${this.selectedItem.name}
</li>
```